pkg/cloudprovider/providers/vsphere/vclib/virtualmachine.go

```diff
@@ -93,6 +93,18 @@ func (vm *VirtualMachine) AttachDisk(ctx context.Context, vmDiskPath string, vol
 		return diskUUID, nil
 	}
 
+	pbmClient, err := NewPbmClient(ctx, vm.Client())
+	if err != nil {
+		glog.Errorf("Error occurred while creating new pbmClient, err: %+v", err)
```

Comment:
Please use a full stop after pbmClient. That's the syntax we are following every where in the code.